### PR TITLE
V1.0.0

### PR DESCRIPTION
**Pull request by generall**

### CODE REVIEW

File: qdrant/v1.0.x/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+For example,
+[http://localhost:6333/telemetry?details_level=6&anonymize=true](http://localhost:6333/telemetry?details_level=6&anonymize=true)
+
+
+## Deactivate telemetry
+
+You can deactivate telemetry by:
+
+- setting the `QDRANT__DISABLE_TELEMETRY` environment variable to `true`
```
Comment (agourlay) [inline qdrant/v1.0.x/telemetry.md:60]: This should be `QDRANT__TELEMETRY_DISABLED` and `telemetry_disabled` according to https://github.com/qdrant/qdrant/pull/1401

The CLI flag `disable_telemetry` is correct though.

Do we want to change the code or the docs for consistency?

Comment (generall) [reply qdrant/v1.0.x/telemetry.md:60]: let's change docs, it's easier

Comment (agourlay) [reply qdrant/v1.0.x/telemetry.md]: https://github.com/qdrant/docs/pull/42/commits/98873220a6a92183c033d216556b381213015ce9

**Comment by agourlay** [comment]
I think we need extra documentation for https://github.com/qdrant/qdrant/pull/1364

**Comment by agourlay** [comment]
Done quickly in https://github.com/qdrant/docs/pull/51

**Pull request merged**